.gitignore

```diff
@@ -99,4 +99,7 @@ typings/
 
 # VS Code debugging
 .vscode/.react/
-apps/*/.vscode/.react/
\ No newline at end of file
+apps/*/.vscode/.react/
+
+# Ignore Gradle build output directory
+**/build
```

Comment:
(Sorry for the followup ask):

I looked at our partner repo react-native-test-app, and noticed they use a slightly more exact regex string, which I think might be slightly more exact / better so we don't unintentionally ignore stuff in the future:
`android/**/build/`
https://github.com/microsoft/react-native-test-app/blob/b7224ef3b9336f5bd853ddbc419f8d2129de9bba/.gitignore#L12

Since we have subdirectories, maybe something like this? I'm not sure if "**" encapsulates if it's a top level directory
```
android/**/build/
**/android/**/build/
```

Reply:
Just looked up some documentation and `**/android/**/build/` is sufficient I think!